tests/test_overclocks.py

```diff
@@ -115,6 +115,99 @@ def test_EBFOverclock(recipe, expected_eut, expected_dur, overclock_handler):
     overclocked = overclock_handler.overclockRecipe(recipe)
     assert overclocked.eut == expected_eut
     assert overclocked.dur == expected_dur
+    
+
+recipe_volcanus = mod_recipe(recipe_ebf, machine="volcanus")
+recipe_samarium = Recipe(
+    "volcanus",
+    "ev",
+    IngCol(
+        Ing("Dephosphated Samarium Concentrate Dust", 1),
+    ),
+    IngCol(
+        Ing("Samarium Oxide Dust", 1),
+        Ing("Gangue Dust", 1),
+    ),
+    514,
+    2,
+    coils="HSS-G",
+    heat=1200,
+)
+
+# Based on in-game measurements
+@pytest.mark.parametrize(
+    "recipe,expected_eut,expected_dur,expected_parallel",
+    [
+        (
+        # No Overclocks (800K over recipe - no bonuses)
+            mod_recipe(recipe_volcanus, user_voltage="mv", coils="cupronickel"),  # 1801K    
+            120 * 0.9,
+            25 / 2.2,
+            1,
+        ),
+        (
+        # No Overclocks (1700K over recipe - one 5% heat bonus)
+            mod_recipe(recipe_volcanus, user_voltage="mv", coils="kanthal"),  # 2701K    
+            120 * 0.9 * 0.95,
+            25 / 2.2,
+            1,
+        ),
+        # 4x voltage for volcanus is 4x parallels
+        (
+            mod_recipe(recipe_volcanus, user_voltage="hv", coils="cupronickel"),  # 1801K    
+            120 * 4 * 0.9,
+            25 / 2.2,
+            4,
+        ),
+        # EBF heat bonuses are applied after parallels are calculated (so still only 4 parallels)
+        (
+            mod_recipe(recipe_volcanus, user_voltage="hv", coils="HSS-G"),  # 5401K
```

Comment:
"HSS-G" should be lowercase since mod_recipe directly modifies value without taking care of cases. Right now, the cases are handled by Recipe's __init__ which is skipped when we use mod_recipe. I should've made it handle cases better.